docs/setup-robusta/installation/index.rst

```diff
@@ -2,8 +2,6 @@
 
 :hide-footer:
 
-.. _install:
```

Comment:
Removing this leads to new warnings: 

<img width="1028" alt="Screenshot 2024-01-12 at 19 24 46" src="https://github.com/robusta-dev/robusta/assets/494087/2c3c4070-fc86-43f2-bdd0-bdb1482f458f">

It's an "anchor" we can link to from anywhere else in the docs. I set it up so that the links will continue to work even if we change the page title.